torch/onnx/__init__.py

```diff
@@ -212,13 +212,13 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
         external_data_format (bool, default False): If True, then the model is exported
             in ONNX external data format, in which case some of the model parameters are stored
             in external binary files and not in the ONNX model file itself. See link for format
-            details: 
+            details:
             https://github.com/onnx/onnx/blob/8b3f7e2e7a0f2aba0e629e23d89f07c7fc0e6a5e/onnx/onnx.proto#L423
             Also, in this case,  argument 'f' must be a string specifying the location of the model.
-            The external binary files will be stored in the same location specified by the model 
+            The external binary files will be stored in the same location specified by the model
             location 'f'. If False, then the model is stored in regular format, i.e. model and
             parameters are all in one file. This argument is ignored for all export types other
-            than ONNX. 
```

Comment:
It seems this file is still auto-formatted, so you may want to check you editor setting. But I think these space changes are small enough that we can include them in this PR.

Reply:
Good call. I had Editor: Format on Save off already, but I turned Editor > Comments: Insert Space off as well, which I believe should fix the problem going forward.